lib/colab.py

```diff
@@ -0,0 +1,52 @@
+from pathlib import Path
+import re
+import subprocess
+import sys
+
+
+try:
+  import google.colab
+  IN_COLAB = True
+except:
+  IN_COLAB = False
+
+
+# library not to install in collab
+COLLAB_EXCLUDE = {"tensorflow", "ipython", "notebook"}
```

Comment:
Were you able to install robotoff with `colab_pip_install`? I don't remember what exactly went wrong but I had a version conflict between one of robotoff's dependencies and libs pre-installed in collab.

Reply:
No problem with robotoff on my side…